src/openrct2/network/Network.cpp

```diff
@@ -310,10 +310,10 @@ class Network
     void Client_Handle_EVENT(NetworkConnection& connection, NetworkPacket& packet);
     void Client_Handle_TOKEN(NetworkConnection& connection, NetworkPacket& packet);
     void Server_Handle_TOKEN(NetworkConnection& connection, NetworkPacket& packet);
-    void Client_Handle_OBJECTS(NetworkConnection& connection, NetworkPacket& packet);
+    void Client_Handle_OBJECTS_LIST(NetworkConnection& connection, NetworkPacket& packet);
     void Client_Handle_SCRIPTS(NetworkConnection& connection, NetworkPacket& packet);
     void Client_Handle_GAMESTATE(NetworkConnection& connection, NetworkPacket& packet);
-    void Server_Handle_OBJECTS(NetworkConnection& connection, NetworkPacket& packet);
+    void Server_Handle_MAPREQUEST(NetworkConnection& connection, NetworkPacket& packet);
```

Comment:
Seems like things should either be unified as `MAP_REQUEST` or `MAPREQUEST`
```suggestion
    void Server_Handle_MAP_REQUEST(NetworkConnection& connection, NetworkPacket& packet);
```

Reply:
I agree, changing it to MAPREQUEST. That slipped on my end.